src/main/java/com/google/api/codegen/py/PythonGapicContext.java

```diff
@@ -250,7 +250,8 @@ private String returnTypeComment(
 
     // parameter types
     contentBuilder.append("Args:\n");
-    for (Field field : this.messages().flattenedFields(method.getInputType())) {
+    for (Field field :
+        removePageTokenFromFields(this.messages().flattenedFields(method.getInputType()), config)) {
```

Comment:
Ahh, I realize that was unclear. What I mean is that the implementation of flattenedFields could be substituted here:

removePageTokenFromFields(method.getInputType().getMessageType().getFields()), config)

And then maybe flattenedFields isn't needed anywhere anymore.